weblate/trans/models/translation.py

```diff
@@ -1429,6 +1429,19 @@ def add_unit(  # noqa: C901
             self.notify_new(request)
         return result
 
+    def notify_deletion(self, unit, user):
+        change = Change(
+            translation=self,
+            action=Change.ACTION_STRING_REMOVE,
+            user=user,
+            target=unit.target,
+            details={
+                "source": unit.source,
+                "target": unit.target,
+            },
+        )
+        change.save(force_insert=True)
```

Comment:
This can be a bit more compact:

```suggestion
        self.change_set.create(
            action=Change.ACTION_STRING_REMOVE,
            user=user,
            target=unit.target,
            details={
                "source": unit.source,
                "target": unit.target,
            },
        )
```